docs/how-to/dss/install-dss-cli.rst

```diff
@@ -0,0 +1,29 @@
+Install DSS
+===========
+
+This guide explains how to install the Data Science Stack (DSS) Command Line Interface (CLI).
+
+Overview
+--------
+
+The DSS CLI is distributed as a snap accessible from the [snap store](https://snapcraft.io/data-science-stack).
```

Comment:
Wrong link you need to add keyword to reuse/links.txt and then reference the keyword.